BaseFilter/sections/specific.txt

```diff
@@ -17390,6 +17390,7 @@ circleofcricket.com#$#.modal-backdrop { display: none !important; }
 circleofcricket.com#$##myModal { display: none !important; }
 circleofcricket.com##div.container-full.add:has(> a[href]:first-child)
 ||circleofcricket.com/assets/adv/
+tvmovie.de#$#.adAnything { min-height: 0px !important }
```

Comment:
Yes, website's style has higher specificity, so current rule doesn't work.

But maybe we could change current rule:
https://github.com/AdguardTeam/AdguardFilters/blob/bde18ca5858f9db5005c6b1be4134b3682d09c7b/GermanFilter/sections/specific.txt#L2733

to something like:
```adblock
tvmovie.de,wunderweib.de##body .adAnything:not(#style_important)
```
So it should works fine in CoreLibs apps too.

Reply:
I thought that maybe the current rule might be modified to:
```adblock
tvmovie.de,wunderweib.de##body .adAnything:not(#style_important)
```
so it wouldn't be necessary to add additional rules and it should works on all platforms.

Reply:
Yes, it works, Chrome considers this as more specific style. Thanks